app/main/views/briefs.py

```diff
@@ -194,6 +195,90 @@ def update_brief_status(brief_id):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/<any(publish, withdraw):action>', methods=['PUT'])
+def update_brief_status_by_action(brief_id, action):
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.framework.status != 'live':
+        abort(400, "Framework is not live")
+
+    action_to_status = {
+        'publish': 'live',
+        'withdraw': 'withdrawn'
+    }
+    if brief.status != action_to_status[action]:
+        brief.status = action_to_status[action]
+
+        validate_brief_data(brief, enforce_required=True)
+
+        audit = AuditEvent(
+            audit_type=AuditTypes.update_brief_status,
+            user=updater_json['updated_by'],
+            data={
+                'briefId': brief.id,
+                'briefStatus': brief.status,
+            },
+            db_object=brief,
+        )
+
+        db.session.add(brief)
+        db.session.add(audit)
+        db.session.commit()
+
+    return jsonify(briefs=brief.serialize()), 200
+
+
+@main.route('/briefs/<int:brief_id>/create-draft-from-withdrawn-brief', methods=['POST'])
```

Comment:
There are two points here:

1. No matter what the restriction is I think the route should be called `/copy`, as that's the actual action
2. I don't see a reason to restrict it to 'withdrawn', so I'd say take any brief and make a 'draft' copy of it.